StylableSwiftUI/Classes/StylistIdentifier.swift

```diff
@@ -36,6 +36,11 @@ public struct StylistIdentifier: Equatable, Hashable {
 
     /// Given the identifier `header/searchBar/title` then the components are `[ "header", "searchBar" ]`
     let path: Path
+
+    /// The theme for this identifier, if any.
+    /// Given the identifier `@dark/header/searchBar/title` then the theme is `@dark`
+    /// Given the identifier `header/searchBar/title` then the theme is `nil`
+    let theme: Theme?
```

Comment:
Just a sanity question - what happens if someone does `header/@searchBar/title` - are we just going to allow that as an insane choice of name for the searchBar component?

Reply:
I don't think that's a problem, but let me write a test for it. The special character (`@`) for the theme is meant as a marker in the identifier that can be extracted when transforming from Sketch file format to Swift. In practice, path components can have any format they want, although some paths might not make much sense. 

I'm thinking "big list of naughty strings"-style inputs - we could write code to make sure it's sturdier but how much more complexity do we want to add for a problem that can be solved by "don't use special characters in path components" entry in the readme?

@cdemetriadis